scrapers_next/oh/committees.py

```diff
@@ -0,0 +1,385 @@
+from spatula import HtmlPage, HtmlListPage, CSS, XPath, SkipItem, URL, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class SenateCommitteeTitleMissingSuffix(Exception):
+    def __init__(self, title):
+        super().__init__(f"Senate committee title is missing expected suffix: {title}")
+
+
+class UnknownJointCommitteeWebsite(Exception):
+    def __init__(self, href):
+        super().__init__(f"Unknown joint committee website: {href}")
+
+
+class UnexpectedMemberTitle(Exception):
+    def __init__(self, title):
+        super().__init__(f"Member title does not follow expected format: {title}")
+
+
+class JointCommittee(HtmlPage):
+    def process_page(self):
+        com = ScrapeCommittee(
+            name=self.input.get("name"),
+            chamber="legislature",
+            classification="committee",
+        )
+
+        com.add_source(self.source.url, note="Committee member listing page")
+
+        for (name, role) in self.get_members():
```

Comment:
Given the signature of `get_members` below...won't this _always_ be an empty list?

Reply:
No, JointCommittee is a base class and the get_members method is overridden by the classes that extend it. I've added a comment and changed the default implementation to raise an error if it's ever called.